hazelcast-integration/docker-compose/src/main/resources/hazelcast-client.xml

```diff
@@ -1,15 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<hazelcast-client xmlns="http://www.hazelcast.com/schema/client-config"
+<hazelcast-client xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                   xsi:schemaLocation="http://www.hazelcast.com/schema/client-config
-                                      http://www.hazelcast.com/schema/client-config/hazelcast-client-config-3.11.xsd"
-                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+                               http://www.hazelcast.com/schema/client-config/hazelcast-client-config-4.0.xsd"
```

Comment:
The executables in this code sample seems to work with 4.0 so bumped the version. Someone needs to verify that it works with docker though.